internal/restapi/providers/v1/handler.go

```diff
@@ -99,13 +99,12 @@ func (h *providersV1HttpService) downloadProviderHandler() http.Handler {
 		providerVersion, providerOS, providerArch := GetProviderInputsFromRequest(r)
 		span.SetAttributes(
 			attribute.String("module.name", providerName),
```

Comment:
Why is module name a span attribute here? 
It does not serve a module, please remove the attribute.

Reply:
missed to update the attribute name, it was additional and was not required, removed it.